src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -145,4 +158,170 @@ void deleteLine() {
         // then
         assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
     }
+
+    /**
+     * Given 지하철 노선 생성을 요청 하고
+     * Given 지하철 역 생성을 요청 하고
+     * When 노선에 등록된 하행 종점 역을 상행 역으로
+     * When 새로 생성한 역을 하행 역으로
+     * When 지하철 구간 등록을 요청 하면
+     * Then 지하철 구간 등록이 성공 한다.
+     */
+    @DisplayName("지하철 구간 등록")
+    @Test
+    void addSection() {
+        // given
+        final ExtractableResponse<Response> createLineResponse = 지하철_노선_생성_요청();
```

Comment:
인수 테스트의 테스트 내부 코드를 조금 더 가독성 좋게 만들어 보는 방향으로 리팩터링 해보면 어떨까요?
구체적으로 어떤 코드를 동작시켜서 검증하고 있는지 보다는 어떤 행위를 했는지를 한눈에 읽을 수 있는 방향으로 리팩터링 해보세요~
예를 들면 응답값들을 모두 변수로 담아두지 말고 아래처럼 변수로 관리해보아도 좋을 것 같아요.

```suggestion
        final Long lineId = 지하철_노선_생성_요청().jsonPath().getLong("id");
```